src/sql/workbench/contrib/query/browser/statusBarItems.ts

```diff
@@ -193,11 +193,15 @@ export class RowCountStatusBarContributions extends Disposable implements IWorkb
 		}
 	}
 
-	private _displayValue(runner: QueryRunner) {
+	private _displayValue(runner: QueryRunner): void {
+		if (!runner.batchSets) {
+			return;
+		}
 		const rowCount = runner.batchSets.reduce((p, c) => {
-			return p + c.resultSetSummaries.reduce((rp, rc) => {
+			const cnt = c.resultSetSummaries ? c.resultSetSummaries.reduce((rp, rc) => {
```

Comment:
Could also do `const cnt = c.resultSetSummaries?.reduce(...) ?? 0` (which I find slightly more readable. Not a huge deal though if you prefer this way)